j2/rules/cap_marketing_campaign_reporting.json

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
We need to talk about this.   Not sure that we need this...  let's review the POLE model.